core/src/main/java/com/sequenceiq/cloudbreak/service/publicendpoint/FreeIPAEndpointManagementService.java

```diff
@@ -77,6 +80,7 @@ private void sendAddDnsARecordRequest(Stack stack, LoadBalancer loadBalancer) {
         request.setHostname(endpoint);
         request.setIp(ip);
         request.setEnvironmentCrn(stack.getEnvironmentCrn());
+        request.setCreateReverse(true);
```

Comment:
when this would be cleaned up from FreeIPA?

Reply:
@hreeve-cloudera are the loadbalancer related domain names added to the freeipa cleanup?

Reply:
Yes, when the stack is terminated. See `ClusterPublicEndpointManagementService.terminate()`.

Since adding the reverse lookup is part of the original registration request, I would assume the current deletion logic would delete both the forward and reverse entries. But we might want to doublecheck that.

Reply:
I see that the reverse DNS entries are cleared. I have created 4 to 5 clusters in the same environment. Only one of the reverse entry is leaked. I strongly suspect I did some force delete.

```
  Record name: 75.6
  PTR record: deha2-gateway.gk1dev.xcu2-8y8x.dev.cldr.work.
```